pom.xml

```diff
@@ -50,8 +54,8 @@
       <url>https://raw.githubusercontent.com/lintool/AnseriniMaven/master/mvn-repo/</url>
     </repository>
     <repository>
-      <id>jitpack.io</id>
-      <url>https://jitpack.io</url>
+      <id>strategicblue</id>
```

Comment:
I don't think we need this?
The artifact is published in Maven central:
https://central.sonatype.com/artifact/blue.strategic.parquet/parquet-floor